tests/testthat/test-retriever.R

```diff
@@ -1,7 +1,107 @@
-library(testthat)
-library(rdataretriever)
+context("regresion tests")
+library(DBI)
+library('RPostgreSQL')
+
 
 test_that("datasets returns some known values", {
   skip_on_cran()
   expect_identical("car-eval" %in% rdataretriever::datasets(), TRUE)
 })
+
+
+test_that("Download the raw portal dataset into './data/'", {
+  portal <- list("3299474", "3299483", "5603981")
+  rdataretriever::download('portal', './data/')
+  for(file in portal)
+  {
+    file_path <- normalizePath(file.path(getwd(), "tests/testthat/data/", file), winslash = "//", mustWork = FALSE)
+    expect_identical((file.info(file_paths)$size == 0)==FALSE, TRUE)
```

Comment:
Do we need to do the `==FALSE` and test against `TRUE` or could we drop `==FALSE` and test against `FALSE`? 

E.g.,

```
expect_identical((file.info(file_paths)$size == 0), FALSE)
```

Or better yet, use the `expect_gt` function (which checks to see if the first value is greater than the second):

```
expect_gt(file.info(file_paths)$size, 0)
```


Same question applies several times below.

Reply:
Where does `file_paths` come from? Is it supposed to be `file_path`? Same question applies below.